web-components/src/components/timepicker/TimePicker.ts

```diff
@@ -291,6 +312,7 @@ export class TimePicker extends LitElement {
     return html `
       <md-combobox
         compact
+        selectWhenInFocus
         class="amPm-combo-box"
```

Comment:
Nit: when typing in this box it uses lowercase, but options are uppercase. Maybe `text-transform:uppercase` will do the trick.